lib/export/cimpl6Exporter.js

```diff
@@ -100,26 +100,26 @@ class Cimpl6Exporter {
     let ns = namespace.replace(/\./, '_');
 
     const hierarchyPath = `${filePath}/${ns}.txt`;
-//console.log("hierarchyPath = "+JSON.stringify(hierarchyPath));  
+//console.log("hierarchyPath = "+JSON.stringify(hierarchyPath));
     mkdirp.sync(hierarchyPath.substring(0, hierarchyPath.lastIndexOf('/')));
     fs.writeFileSync(hierarchyPath, formattedFile);
-//console.log("DONE WRITING FILE");  
+//console.log("DONE WRITING FILE");
   }
 
-  /*exportValueSetFileToNamespace(formattedFile, namespace, filePath) {
+  exportValueSetFileToNamespace(formattedFile, namespace, filePath) {
```

Comment:
It's fine to uncomment the function declaration, as long as we're not actually using it anywhere, but since the VS exporter hasn't been tested at all -- and we now know there *are* bugs in it -- I think we should at least add a comment indicating that it hasn't been tested and may not be correct.

Reply:
yeah, i'll comment it again.